.buildkite/hooks/post-command

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [[ "$BUILDKITE_AGENT_NAME" =~ ^bk-agent ]]; then
-  echo "Pipeline file triggered from outside the kibana executors, skipping post_command"
-else
-  .buildkite/scripts/lifecycle/post_command.sh
-fi
+source .buildkite/scripts/lifecycle/post_command.sh
```

Comment:
Is this going to work if SKIP_NODE_SETUP is set?  There's another reference to `ts-node .buildkite/scripts/lifecycle/print_agent_links.ts || true` in post_command.sh.

Reply:
Good question. I thought the `|| true` will avoid errors coming from there, but I'll separate that call better depending on `SKIP_NODE_SETUP`

Reply:
Ah you're right.  I'll leave it up to you, I'm okay with how it is.

Reply:
I'll test to see if it works as expected still